mon-pix/tests/acceptance/challenge-item_test.js

```diff
@@ -78,8 +78,8 @@ module('Acceptance | Displaying a challenge of any type', function (hooks) {
 
               test('should enable input and buttons', async function (assert) {
                 // then
-                assert.notOk(find('.challenge-actions__action-skip').getAttribute('disabled'));
-                assert.notOk(find('.challenge-actions__action-validate').getAttribute('disabled'));
+                assert.ok(find('.challenge-actions__action-skip').getAttribute('aria-disabled').includes('false'));
```

Comment:
Je propose d'utiliser les méthodes d'ember testing library plutôt :) 
```suggestion
                assert.dom(screen.getByRole('button', { name : "Je passe et je vais à la prochaine question" }).hasAttribute('aria-disabled', 'false');
```

Reply:
On voit ça dans un autre ticket